docs/configuration/install.md

```diff
@@ -90,11 +90,12 @@ We RECOMMEND at least 100 Mbit/sec. for the web server WAN connection.
 For best results we RECOMMEND:
 
   * Debian GNU Linux 8 / Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
-  * MySQL 5.5 / MariaDB
-  * PHP 5.6+
-  * Apache 2.2+ with mod_php
+  * MySQL 5.6+
+  * MariaDB 10+
+  * PHP 7.1
+  * Apache 2.4+ with mod_php
   * ImageMagick 6.8+
-  * php5-gd, php5-xsl, php5-mysql
+  * php7.1-gd, php7.1-xsl, php7.1-mysql
```

Comment:
Since GD comes with PHP, can we remove it from this line?

Reply:
I would prefer to get rid of "open end version" recommendations completely. E.g. instead of **MariaDB 10+** I would prefer **MariaDB 10.0 - 10.2** or **MariaDB 10.1**. If in doubt I would always use the exact version of our test system.

Reply:
I think php-xml should be mentioned, too.

Reply:
I am not sure about gd. I think the embedded gd support will be active in most distributions per default. But I also think it would be possible to deactivate gd support, which users should not do, so I am fine with keeping it in this list.